lib/src/util/map_util.dart

```diff
@@ -78,6 +130,6 @@ Map<String, dynamic> newStyleFromProps(Map props) {
   return existingStyle == null ? <String, dynamic>{} : new Map.from(existingStyle);
 }
 
-SplayTreeSet _validDomProps = new SplayTreeSet()
+HashSet _validDomProps = new HashSet()
```

Comment:
Just want to verify that this is backwards-compatible with `addProps(copyUnconsumedProps())` - since that old pattern also uses this variable.

Reply:
I can't find any reason why it wouldn't be! It's used pretty limitedly, and the only methods the object needs are `addAll` and `contains`. There are also still unit tests (and manual tests that I had done), and there haven't been any red flags!